### PR TITLE
Feature to enable or disable SSL for Postgres

### PR DESCRIPTION
**Pull request by vrtata**
**Description of the change**

The change implements an environment variable which enables switching on SSL mode for database connections during deployment.

**Benefits**

With the increase in usage of cloud databases, it is required by organization to connect to databases over SSL. In some cases it is mandatory.

**Possible drawbacks**

None

**Applicable issues**

Please refer to the issue https://github.com/bitnami/charts/issues/6217 titled "[bitnami/keycloak] unable to specify sslmode for external PostgreSQL database"

**Additional information**

### CODE REVIEW

**Comment by rafariossaa** [comment]
Hi,
  Thanks for this providing this functionality.
  I am doing the internal changes to acomodate this before merging it.

**Comment by rafariossaa** [comment]
Could you add the new environment variables  to the README ?

File: 13/debian-10/rootfs/opt/bitnami/scripts/keycloak-env.sh [hunk 1]
```diff
@@ -147,6 +149,8 @@ KEYCLOAK_DATABASE_PASSWORD="${KEYCLOAK_DATABASE_PASSWORD:-"${DB_PASSWORD:-}"}"
 export KEYCLOAK_DATABASE_PASSWORD="${KEYCLOAK_DATABASE_PASSWORD:-}"
 KEYCLOAK_DATABASE_SCHEMA="${KEYCLOAK_DATABASE_SCHEMA:-"${DB_SCHEMA:-}"}"
 export KEYCLOAK_DATABASE_SCHEMA="${KEYCLOAK_DATABASE_SCHEMA:-public}"
+KEYCLOAK_DATABASE_SSL="${KEYCLOAK_DATABASE_SSL:-"${DB_SSL:-}"}"
```
Comment (rafariossaa) [inline 13/debian-10/rootfs/opt/bitnami/scripts/keycloak-env.sh:152]: Please, could you rename this environment variable as `KEYCLOAK_DATABASE_TLS` ?. This is for coherence with the rest of containers.

**Review by rafariossaa — CHANGES_REQUESTED** [review]
Sorry, I realized about a couple of details.

Could you take care of them? 
One is renaming the new environment variable, and the other is adding a validation for the new environment variable values, for example:

```diff
diff --git a/images/generic/lib/libkeycloak.sh b/images/generic/lib/libkeycloak.sh
index a9c1a001085f9..2217266b7c3da 100644
--- a/images/generic/lib/libkeycloak.sh
+++ b/images/generic/lib/libkeycloak.sh
@@ -65,7 +65,7 @@ keycloak_validate() {
     check_allowed_port KEYCLOAK_HTTP_PORT
     check_allowed_port KEYCLOAK_HTTPS_PORT
 
-    for var in KEYCLOAK_CREATE_ADMIN_USER KEYCLOAK_ENABLE_TLS KEYCLOAK_ENABLE_STATISTICS; do
+    for var in KEYCLOAK_CREATE_ADMIN_USER KEYCLOAK_ENABLE_TLS KEYCLOAK_ENABLE_STATISTICS KEYCLOAK_DATABASE_TLS; do
         if ! is_true_false_value "${!var}"; then
             print_validation_error "The allowed values for $var are [true, false]"
         fi
```

**Comment by vrtata** [comment]
Sure. I will make the necessary changes and submit a new pull request

**Review by rafariossaa — APPROVED** [review]
LGTM.

**Pull request merged**